README.md

```diff
@@ -162,6 +162,15 @@ The significant keys in the YAML frontmatter are:
 
 There is currently a small bug with new page creation - if you create a new page and link it up via subnav data while the server is running, it will report an error saying the page was not found. This can be resolved by restarting the server.
 
+#### Adding New Pages Under `content/docs`
+
+Due to the way we handle old `/docs` redirects, you need to explicitly add your new page to the `DEFINED_DOCS_PAGES` array in `pages/docs/[[...page]].jsx` in addition to creating the MDX file and updating `data/docs-nav-data.json`. For example, if you created an MDX file at the path `content/docs/why-terraform-is-awesome.mdx`, you would add the string `'why-terraform-is-awesome'` to the `DEFINED_DOCS_PAGES` array.
```

Comment:
I love this - it's great. Do you think we should add one more sentence being like "you don't need to do this for any other directories - just this one"